apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -49,7 +49,7 @@ export default class DetailViewContents extends React.Component {
 
   state = {
     status: this.props.applicationData.status,
-    notes: this.props.applicationData.notes
+    notes: this.props.applicationData.notes || "Google doc rubric completed: Y/N\nTotal points:\n(If interviewing) Interview notes completed: Y/N\nAdditional notes:"
```

Comment:
Do we need to expand the Notes box to include enough lines for this field?

Reply:
Also - should we have a link to the google doc rubric in the detail view?

Reply:
I expanded the notes box for this text in a [previous PR](https://github.com/code-dot-org/code-dot-org/pull/19055)

Reply:
It probably would make sense to have a link - do you want to include that with scoring? (I'm not sure what the page will look like once that is in or where it would make the most sense to put the link)